### ISSUE TITLE
 ImportError: libgthread-2.0.so.0: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by Badrabu** [roboflow/notebooks#269]
Im using  Sagemaker studio lab, its a managed notebook and still face the following error 

~/.conda/envs/default/lib/python3.9/site-packages/ultralytics/yolo/engine/predictor.py in <module>
     30 from pathlib import Path
     31 
---> 32 import cv2
     33 
     34 from ultralytics.nn.autobackend import AutoBackend

ImportError: libgthread-2.0.so.0: cannot open shared object file: No such file or directory

**Comment by turbotimon** [comment]
In my case (on NixOS, but should work for other distros too) it was enough to install the headless version:

```
pip uninstall opencv-python
pip install opencv-python-headless
```

**Comment by SkalskiP** [comment]
Yup @turbotimon and @Badrabu the error you're encountering, `ImportError: libgthread-2.0.so.0: cannot open shared object file: No such file or directory`, typically occurs in scenarios involving GUI applications or libraries on systems that lack the necessary GUI support libraries. This can happen when using the GUI version of OpenCV on a non-GUI (headless) operating system.
